opentelemetry-http/src/lib.rs

```diff
@@ -250,6 +262,32 @@ mod tests {
         )
     }
 
+    #[test]
+    fn http_headers_get_all() {
+        let mut carrier = http::HeaderMap::new();
+        carrier.append("headerName", HeaderValue::from_static("value"));
+        carrier.append("headerName", HeaderValue::from_static("value2"));
+        carrier.append("headerName", HeaderValue::from_static("value3"));
```

Comment:
Ignore this, since the GitHub UI doesn't allow me to delete comments right now 🤷🏼‍♂️